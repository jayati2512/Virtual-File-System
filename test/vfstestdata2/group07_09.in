createvfs vfs 1024
deletedir /dir
mountvfs
mountvfs vfs
deletedir /dir
makedir dir
makedir / dir
deletedir dir
copyfile dir /input.txt
