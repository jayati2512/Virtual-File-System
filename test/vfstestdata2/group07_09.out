createvfs_SUCCESS
deletedir_FAILURE VFS_NOT_MOUNTED 
mountvfs_FAILURE VFS_INSUFFICIENT_ARGUMENTS 
mountvfs_SUCCESS
deletedir_FAILURE CANNOT_FIND_SPECIFIED_DIR 
makedir_FAILURE VFS_INSUFFICIENT_ARGUMENTS 
makedir_SUCCESS
deletedir_SUCCESS
copyfile_FAILURE CANNOT_COPY_DIR_TO_FILE 
